modules/navier_stokes/test/tests/finite_volume/pwcns/channel-flow/gold/2d-transient-gas_out.csv

```diff
@@ -0,0 +1,5 @@
+time,inlet-p,outlet-temp,outlet-u,solid-temp
+0,0,0,0,0
+1,8.6557148893077,106.39295004178,-2.9848928309229,104.9552224657
```

Comment:
Still weird with the PP type change. Is there actually an inflow at this first time step or is this still a postprocessing artifact?

Reply:
my bet is inflow during the transient. Lemme send you a gif

Reply:
this is already discussed in another conversation